src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
>If I am not mistaken easyrgb and lindbloom calculators adapt the primaries, so I think chromatic adaptation is the way to go.

Yeah, we should look into that. They are a bit more specialized, though. The easyrgb is more or less a black box, but we can poke it from the outside. Lindbloom's calculator is all JavaScript, so the source can easily be inspected through the browser.

>One issue I currently see is the the AdaptFrom does not check if the source and destination white points are equal and will cause D65 -> D65 changes. I guess we should add a test for that.

We could do the `TypeId` comparison there, as well. It's may give false negatives, but only when different types represents the same values. I think that's acceptable at the moment.